Makefile

```diff
@@ -0,0 +1,43 @@
+.PHONY: build test clean update fmt docker run
+
+GO=CGO_ENABLED=1 GO111MODULE=on go
+
+MICROSERVICE=rfid-llrp-inventory
+
+.PHONY: build test clean fmt docker run
+
+VERSION=$(shell cat ./VERSION 2>/dev/null || echo 0.0.0)
+GIT_SHA=$(shell git rev-parse HEAD)
+
```

Comment:
Can we simplify it to use just one `sed` instead of `cat`, `sed`, `grep`, and `awk`?

```makefile
# This pulls the version of the SDK from the go.mod file
SDKVERSION=$(shell sed -En 's|.*github.com/edgexfoundry/app-functions-sdk-go (v[\.0-9a-zA-Z-]+).*|\1|p' go.mod)
```

Reply:
Sure.